src/frontend/src/app/app.component.html

```diff
@@ -38,6 +38,23 @@
       </div>
     </div>
       <app-playlist-box *ngFor="let playlist of playlists$ | async" [playlist]="playlist"></app-playlist-box>
+      <p *ngIf="(playlists$ | async)?.length === 0" class="has-text-grey has-text-centered">No playlists</p>
+    </div>
+    <hr>
+    <div class="box">
+    <div class="is-flex is-justify-content-space-between">
+      <p class="subtitle">Songs</p>
+      <div class="buttons has-addons">
+        <button class="button is-outlined is-success" title="Remove completed from list" (click)="deleteCompleted()">
```

Comment:
I bet this delete completeds songs, but also playlists. But can be kept and fixed in the future.